### PR TITLE
Add possibility for extra envs

### PR DESCRIPTION
**Pull request by pixiono**
Since we have a cluster in an environment without Internet access, we needed a proxy to reach Azure AD. For this we need appropriate environment variables.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
Hello @pixiono could you also update the CHANGELOG.md file https://github.com/giantswarm/oauth2-proxy-app/blob/main/CHANGELOG.md ?

Did you test your changes with multiple entries and add an example in the `values.yaml` file as a comment?

**Comment by ubergesundheit** [comment]
Please also update the `values.schema.json` file next to the values.yaml. Thanks!

**Comment by pixiono** [comment]
@ubergesundheit thanks for your reply, I added the requested information. 

>  you test your changes with multiple entries and add an example in the values.yaml file as a comment?
Yes, I did. I also added the example in the values.yaml

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.8.0] - 2023-02-01
```
[the remark this answers is not in the document]
Comment (pixiono) [reply CHANGELOG.md]: [Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

File: CHANGELOG.md [hunk 2]
```diff
@@ -143,7 +153,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Push `g8s-oauth2-proxy` chart into `control-plane` catalog instead of quay.io.
 - Push `g8s-oauth2-proxy` app CRs into `<provider>-app-collection` repository.
 
-[Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...HEAD
+[Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.8.0...HEAD
```
[the remark this answers is not in the document]
Comment (pixiono) [reply CHANGELOG.md]: [Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

File: helm/oauth2-proxy/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: oauth2-proxy
 description: A reverse proxy that provides authentication with Google, Azure, OpenID Connect and many more identity providers
 home: https://github.com/giantswarm/oauth2-proxy-app
-version: "2.7.1"
```
[the remark this answers is not in the document]
Comment (pixiono) [reply helm/oauth2-proxy/Chart.yaml]: [Done](https://github.com/giantswarm/oauth2-proxy-app/pull/103/commits/39defa1f98968f139dacaaa0056ce80d3582a025)

File: CHANGELOG.md [hunk 4]
```diff
@@ -144,6 +153,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Push `g8s-oauth2-proxy` app CRs into `<provider>-app-collection` repository.
 
 [Unreleased]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...HEAD
+[2.8.0]: https://github.com/giantswarm/oauth2-proxy-app/compare/v2.7.1...v2.8.0
```
Comment (ubergesundheit) [inline CHANGELOG.md:156]:
```suggestion
```

**Comment by ubergesundheit** [comment]
Thank you for your contribution!

**Comment by ubergesundheit** [comment]
We just released your contribution as https://github.com/giantswarm/oauth2-proxy-app/releases/tag/v2.8.0

it should be available for installation soon

**Pull request merged**